xray/commands/curation/audit.go

```diff
@@ -305,28 +306,36 @@ func (ca *CurationAuditCommand) CommandName() string {
 }
 
 func (ca *CurationAuditCommand) SetRepo(tech coreutils.Technology) error {
-	switch tech {
-	case coreutils.Npm:
-		configFilePath, exists, err := rtUtils.GetProjectConfFilePath(rtUtils.Npm)
-		if err != nil {
-			return err
-		}
-		if !exists {
-			return errorutils.CheckError(errors.New("no config file was found! Before running the npm command on a " +
-				"project for the first time, the project should be configured using the 'jf npmc' command"))
-		}
-		vConfig, err := rtUtils.ReadConfigFile(configFilePath, rtUtils.YAML)
-		if err != nil {
-			return err
-		}
-		resolverParams, err := rtUtils.GetRepoConfigByPrefix(configFilePath, rtUtils.ProjectConfigResolverPrefix, vConfig)
-		if err != nil {
-			return err
-		}
-		ca.setPackageManagerConfig(resolverParams)
+	projType := cmdUtils.TechToProjectType(tech)
+	if projType == -1 {
+		return errorutils.CheckErrorf(errorTemplateUnsupportedTech, tech.ToString())
+	}
+	switch projType {
+	case rtUtils.Npm, rtUtils.Yarn:
+		return ca.setRepoByProjectType(projType)
 	default:
 		return errorutils.CheckErrorf(errorTemplateUnsupportedTech, tech.ToString())
 	}
+}
+
+func (ca *CurationAuditCommand) setRepoByProjectType(projectType rtUtils.ProjectType) error {
+	configFilePath, exists, err := rtUtils.GetProjectConfFilePath(projectType)
+	if err != nil {
+		return err
+	}
+	if !exists {
+		return errorutils.CheckError(errors.New("no config file was found! Before running the npm command on a " +
+			"project for the first time, the project should be configured using the 'jf npmc' command"))
```

Comment:
Note that there are two different config commands fornpm and yarn. Let's ensure that this message includes the right command,